src/components/capture/Interval/index.tsx

```diff
@@ -57,31 +41,17 @@ function CaptureInterval({ readOnly }: CaptureIntervalProps) {
     const formActive = useFormStateStore_isActive();
     const formStatus = useFormStateStore_status();
 
-    const lastIntervalChar = interval?.at(-1) ?? '';
-    const singleUnit = ['h', 'i', 'm', 's'].some(
-        (symbol) => lastIntervalChar === symbol
-    );
-
-    const [unit, setUnit] = useState(singleUnit ? lastIntervalChar : '');
-    const [input, setInput] = useState(
-        singleUnit ? interval?.substring(0, interval.length - 1) : interval
-    );
+    const [input, setInput] = useState(interval ?? '');
 
     const loading = formActive || formStatus === FormStatus.TESTING_BACKGROUND;
 
-    const errorsExist = useMemo(() => {
-        const intervalErrorsExist =
-            input &&
-            hasLength(input) &&
-            !POSTGRES_INTERVAL_RE.test(input) &&
-            !CAPTURE_INTERVAL_RE.test(input);
-
-        return Boolean(
-            unit === 'i'
-                ? intervalErrorsExist
-                : intervalErrorsExist && !NUMERIC_RE.test(input)
-        );
-    }, [input, unit]);
+    const errorsExist = useMemo(
+        () =>
+            Boolean(
+                input && hasLength(input) && !CAPTURE_INTERVAL_RE.test(input)
+            ),
```

Comment:
We have basically the same test in 3 places - we should just make this a general function in the time utils.

Reply:
Sweeping through the code to see where this condition appears three times. As of right now, I do not see where this is occurring.